### ISSUE TITLE
`-a/--all` does not work w/o `hub`

### ISSUE THREAD

**Issue by vladdeSV** [wilzbach/splitup-prs#1]
I ran `splitup-prs -p "[swedish]" -f --all`, and got the message at the end: `step-by-step executing is activated. use -a/--all to run for all remaining.`

This was tested on a Raspberry Pi, Raspian without `hub` installed

**Comment by wilzbach** [comment]
Dear brave alpha-warrior,

thanks a lot for your feedback :)
I think I found your issue - the following prints "bash":

``` sh
STEP_BY_STEP=0
if [ $STEP_BY_STEP  ] ; then
    echo "bash"
fi
```

Sorry about that - I just pushed it into the master branch (it got a big diff, because for some stupid reason I used tabs in the beginning). Let me know if it fixed your issue.

https://github.com/wilzbach/splitup-prs/commit/040b00b7ca3f2eeef88e775f97445546d9a9d14a#diff-b8b0c417b635d03a53b82dcb64ce3c98R115

Btw as far as I can observe the step-by-step worked for you?
Was there anything that bothered you or could be improved?

**Comment by vladdeSV** [comment]
I'll do some tests in the coming time.

I had no problems with the application, other than it didn't do all PR's. I simply ran the same command four times and it did what I expected it to do :smile:

**Comment by wilzbach** [comment]
> I'll do some tests in the coming time.

Yup & don't hesitate to ping me in case you run into issues again ;-)

> I had no problems with the application, other than it didn't do all PR's. I simply ran the same command four times and it did what I expected it to do

Glad to hear that - yep it's a stupidly simple helper tool :P
